Content.Server/Doors/Systems/DoorSystem.cs

```diff
@@ -280,5 +282,22 @@ protected override void CheckDoorBump(DoorComponent component, PhysicsComponent
             }
         }
     }
-}
 
+    private void OnMapInit(EntityUid uid, DoorComponent door, MapInitEvent args)
+    {
+        // Copy access from the door to the door electronics at the start.
+        if (!_containerSystem.TryGetContainer(uid, "board", out var boardContainer))
+            return;
+
+        if (!TryComp<AccessReaderComponent>(uid, out var access))
+            return;
+
+        foreach (var entity in boardContainer.ContainedEntities)
+        {
+            if (TryComp<AccessReaderComponent>(entity, out var boardAccess))
+                boardAccess.AccessLists = access.AccessLists;
+        }
+
+        EntityManager.RemoveComponent<AccessReaderComponent>(uid);
+    }
```

Comment:
this is a workaround to just tweaking the airlock prototypes to remove their innate access and put door boards replacements

Reply:
Yes, as I talked here https://github.com/space-wizards/space-station-14/pull/17778#issuecomment-1616680258 , I can't find any good solution how to move AccessReaderComponent from airlock prototypes to their door electronics.